app/Views/Templates/components/forms/chip/index.blade.php

```diff
@@ -0,0 +1,57 @@
+@props([
+    'type' => '', //status, milestone, effort, priority, user, sprint
+    'parentId' => '',
+    'bgColor' => '',
+    'quickaddOption' => false,
+    'quickaddPostUrl' => '',
+    'labelText'=> '',
+    'selectedKey' => '',
+    'options' => [],
+    'extraClass' => '',
+])
+
+<div {{ $attributes->merge([ 'class' => '' ]) }}>
+    <div class="dropdown ticketDropdown {{ $type }}Dropdown colorized show px-2 py-0.5 ml-1.5 {{ $extraClass }}">
```

Comment:
Seems this is the old dropdown code, we should use our dropdown component for this